hackathon_site/review/jinja2/review/send-decisions.html

```diff
@@ -0,0 +1,53 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}SEND DECISION EMAILS TO PARTICIPANTS{% endblock %}
+{% block form_attrs %}enctype="multipart/form-data"{% endblock %}
+
+{% set form_action=url("admin:send-decision-emails") %}
+
+{% block form_description %}
+    {% if form.non_field_errors %}
```

Comment:
This should be `{% if form.non_field_errors() %}`, and is a mistake in all the other jinja templates as well. `form.non_field_errors` is a function, and `bool(<function>)` is always True. I'm guessing this was originally introduced when using a django template as an example, which doesn't need explicit function calls.